### ISSUE TITLE
openssl speed invalid machine command

### ISSUE THREAD

**Issue by paulbruedgam** [open-quantum-safe/openssl#7]
When I'm running the version from today on Debian 8, I got with "openssl speed oqskex_lwe_frodo_recommended" the output:

Doing OQS KEX LWE Frodo recommended Alice 0's for 10s: Invalid machine command

All other oqskex cipher will run.

**Comment by dstebila** [comment]
Can you provide some more information about the environment your running in?  Specifically exact Debian version, processor, and compiler version?  We'd need that before we can hope to diagnose the error.

**Comment by paulbruedgam** [comment]
Of course. My system has a Debian 8.6 with gcc (Debian 4.9.2-10) 4.9.2. on x86_64.

**Comment by smashra** [comment]
When I ran I got the following:

cat /etc/debian_version 
8.7

 gcc --version
gcc (Debian 4.9.2-10) 4.9.2
Copyright (C) 2014 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

apps/openssl speed oqskex_lwe_frodo_recommended
Doing OQS KEX LWE Frodo recommended Alice 0's for 10s: 13340 OQS KEX LWE Frodo recommended Alice 0 in 9.99s
Doing OQS KEX LWE Frodo recommended Bob's for 10s: 9428 OQS KEX LWE Frodo recommended Bob in 10.00s
Doing OQS KEX LWE Frodo recommended Alice 1's for 10s: 79004 OQS KEX LWE Frodo recommended Alice 1 in 10.00s
OpenSSL 1.0.2k-dev  xx XXX xxxx
built on: reproducible build, date unspecified
options:bn(64,64) rc4(16x,int) des(idx,cisc,16,int) aes(partial) idea(int) blowfish(idx) 
compiler: gcc -I. -I.. -I../include  -DOPENSSL_THREADS -D_REENTRANT -DDSO_DLFCN -DHAVE_DLFCN_H -Wa,--noexecstack -m64 -DL_ENDIAN -O3 -Wall -DOPENSSL_IA32_SSE2 -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_BN_ASM_GF2m -DRC4_ASM -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DMD5_ASM -DAES_ASM -DVPAES_ASM -DBSAES_ASM -DWHIRLPOOL_ASM -DGHASH_ASM -DECP_NISTZ256_ASM
                                 Alice0    Alice0/s          Bob      Bob/s          Alice1   Alice1/s
oqskex (lwe_frodo_recommended)     0.748876ms      1335.3   1.060670ms      942.8      0.126576ms     7900.4
